tools/run-task.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+# This temporary script runs an arbitrary task using an existing task defintion
+# in a new container, using ECS RunTask. This script will be replaced by
+# features built into the GDS CLI.
+
+set -eu
+
+OPTIND=1 # Reset in case getopts has been used previously in the shell.
+
+cluster="task_runner"
+govuk_env="test"
+
+function show_help {
+  echo "Usage: cd govuk-infrastructure && gds aws govuk-tools-internal-admin ./tools/run-rake-task -a frontend rake db:migrate"
+}
+
+while getopts :a:c:e:h opt; do
+    case $opt in
+      h) show_help && exit 0;;
+      a)
+        application=${OPTARG} >&2
+        ;;
+      e)
+        govuk_env=${OPTARG:-$govuk_env} >&2
+        ;;
+      :)
+        echo "Option -$OPTARG requires an app as the argument. e.g. -a frontend" >&2
+        exit 1
+        ;;
+    esac
+done
+
+shift $((OPTIND-1))
+[ "${1:-}" = "--" ] && shift
+command=$@
+
+if [ -z "${command}" ]; then echo "[Error] You must set the application arg" && show_help && exit 1; fi
+if [ -z "${command}" ]; then echo "[Error] You must set provide a command" && show_help && exit 1; fi
+
+echo "[Warning] This is a temporary tool for use by the replatforming team."
+
+root_dir=$(pwd)
+
+app_dir="$root_dir/terraform/deployments/apps/$application"
+env_dir="$root_dir/terraform/deployments/govuk-test"
+
+echo "Fetching task_definition_arn from Terraform statefile in $app_dir"
+cd ${app_dir}
+terraform init >/dev/null
+task_definition_arn=$(terraform output task_definition_arn)
+
+echo "Fetching network_config from Terraform statefile in $env_dir"
+cd ${env_dir}
+terraform init >/dev/null
+private_subnets=$(terraform output -json private_subnets)
+security_groups=$(terraform output -json $application'_security_groups')
+network_config='awsvpcConfiguration={subnets='$private_subnets',securityGroups='$security_groups',assignPublicIp=DISABLED}'
```

Comment:
Consider using one double quoted string here for readability rather than a series of single-quoted strings, something like:

```suggestion
network_config="awsvpcConfiguration={subnets=$private_subnets,securityGroups=$security_groups,assignPublicIp=DISABLED}"
```

Reply:
Cool, I've done that 👍